yt_dlp/extractor/youtube.py

```diff
@@ -100,6 +100,17 @@ def parse_qs(url):
         },
         'INNERTUBE_CONTEXT_CLIENT_NAME': 67,
     },
+    'web_studio': {
+        'INNERTUBE_API_KEY': 'AIzaSyAO_FJ2SlqU8Q4STEHLGCilw_Y9_11qcW8',
+        'INNERTUBE_HOST': 'studio.youtube.com',
```

Comment:
`INNERTUBE_HOST` probably not needed since this key isn't restricted to a domain

Reply:
the real key used by studio.youtube.com is `AIzaSyBUPetSUmoZL-OhlxA7wSac5XinrygCqMo`

Reply:
Client renamed to 'web_creator' and host removed. Seems to work well.